store/SPEC_STANDARD.md

```diff
@@ -0,0 +1,110 @@
+## What is a store specification document?
+
+A store specification document is a design document describing a particular store feature such as a store interface or implementation that it is expected to be used (or already in use) by the Cosmos SDK. This also includes other store-related features such as caches and wrappers.
+
+## Sections
+
+An store specification document consists of, synopsis, overview and basic concepts, technical specification, history log, and copyright notice. All top-level sections are required. References should be included inline as links, or tabulated at the bottom of the section if necessary.  Included sub-sections should be listed in the order specified below. 
```

Comment:
"A store specification"

consists of:
* a synopsis
* overview
* basic concepts
* technical specification
* history/version log
* copyright notice